deployment/deploy-presidio.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 REGISTRY=${1:-mcr.microsoft.com}
 TAG=${2:-latest}
-helm install --name redis stable/redis --set usePassword=false,rbac.create=true --namespace presidio-system --wait
-helm install --name presidio-demo --set registry=$REGISTRY,tag=$TAG ../charts/presidio --namespace presidio
+helm install redis stable/redis --set usePassword=false,rbac.create=true --namespace presidio-system --wait
```

Comment:
perhaps add in docs that we support now only helm3 (which is why this was changed)

Reply:
done

Reply:
actually I reverted these changes. we can move to helm 3 on another pr